eahub/templates/eahub/about.html

```diff
@@ -14,83 +14,129 @@
-            <li>Vaidehi Agarwalla</li>
-            <li>Eve McCormick</li>
-            <li>Jo Duyvestyn</li>
-            <li>Katie Gertsch</li>
-            <li>Liz Aderhold</li>
-            <li>Emma Caplan</li>
-            <li>Rebecca Baron</li>
-            <li>David Vatousios</li>
-            <li>Mac Jordan</li>
-            <li>James Faville</li>
-            <li>Pim Kang</li>
-            <li>Riley Peebles</li>
-            <li>Avi Iyer</li>
-            <li>and Marcin Kowrygo</li>
+          <li><a href="https://eahub.org/profile/catherine-low/">Catherine Low</a> (CEA): Group management</li>
+          <li><a href="https://eahub.org/profile/michal-trzesimiech/">Michael Trzesiemich</a>: Operations</li>
+          <li><a href="https://eahub.org/profile/annie-alexander-barnes/">Annie Alexander-Barnes</a>: Communications</li>
+          <li><a href="https://eahub.org/profile/balazs-rapi/">Balázs Rapi</a>: Operations</li>
+          <li>Haneesha Puruchuri</li>
         </ul>
+        <p>We are also grateful for the feedback of several volunteers and advisors.</p>
 
-        Data Privacy Officer: Victor Yunenko
 
-        <h4 class="mt-4"> Feedback </h4>
+        <h2>History</h2>
+        <p>The EA Hub was originally created by Tom Ash in 2014 to host both his own projects (beginning with the <a href="https://www.rethinkprojects.org/easurvey">EA Survey</a> and the public <a href="https://eahub.org/profiles/">EA Profiles</a> that came out of it) and other independent ones. Tom's projects later moved to their own non-profit, .impact Focus Projects, which was <a href="https://forum.effectivealtruism.org/posts/a3sScjxdgNTCseBMJ/impact-is-now-rethink-charity">renamed Rethink Charity</a> in 2017. The Hub is now fiscally sponsored by <a href="https://rtcharity.org/">Rethink Charity</a>, and run by our team of volunteers. </p>
+
+        <p>The Hub was relaunched in 2019 by a team of staff and volunteers, and we would like to recognize the efforts of the following people who made this possible:</p>
+        <ul>
+          <li>Richenda Herzig: Product Manager and Illustrator</li>
+          <li>Michal Trezsimiech: Project Manager</li>
+          <li>Manoj Nathwani: Tech Lead/Back-End Developer</li>
+          <li>Taymon Beal: Tech Lead/Back-End Developer</li>
+          <li>Sebastian Becker: Front-End Developer</li>
+          <li>Nadia Williams: Illustrator</li>
+          <li>Piotr Cieszynski: Tester</li>
+          <li>Alexander Herwix: Advisor</li>
+        </ul>
+
+        <h2>Get involved</h2>
+        <h4>Volunteer</h4>
+        <p>We are always <a href="http://bit.ly/eahub-openroles">looking for volunteers</a> to help build the Hub, in both technical and non-technical roles. If you’re interested in tech roles, please <a href="https://docs.google.com/forms/d/1DH5mNALNNe79Kbx8akGi56TpvrtaepAEaKsu96jZ-Ks/edit">sign up</a> before starting to work on an issue in our Repo so that we can direct you to the most pressing or suitable tasks.</p>
```

Comment:
And well rel=nofollow everywhere else :)

Probably expect the forum and rethink charity, as they don't add it for our links